fitbenchmarking/utils/fitbm_result.py

```diff
@@ -4,6 +4,8 @@
 
 import numpy as np
 
+LOCAL_MIN_TOL = 1e-2
```

Comment:
I wonder if even this is too tight?  For example, the local minima for the gaussian tests are coming up as non local minima.  I think we should raise this to `1e-1`, but we also need to make sure it's clear to the user what we're telling them here (although the same is true for all the tables, so I don't think we need to do that here).

Reply:
I've set the tolerance to be `1e-1` but not reported this to the users yet. As you said, more information about the tables should be added but I don't think this is the place to do that